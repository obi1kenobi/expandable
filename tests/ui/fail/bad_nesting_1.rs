#[expandable::expr]
macro_rules! bad_nesting_example {
    ( $( $a:ident )? ) => { $( $a )* }
}

fn main() {}
