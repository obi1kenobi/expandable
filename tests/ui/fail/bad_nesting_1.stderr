error: Metavariable `a` must be repeated with `?` nesting. It is repeated with `*`
 --> tests/ui/fail/bad_nesting_1.rs:4:33
  |
4 |     ( $( $a:ident )? ) => { $( $a )* }
  |                                 ^
